src/main/scala-2.11/zio/prelude/TraversableVersionSpecific.scala

```diff
@@ -20,6 +20,7 @@ trait TraversableVersionSpecific {
   object DeriveCanBuildFrom {
     implicit def default[F[+_]](implicit bf: CanBuildFrom[F[Any], Any, F[Any]]): DeriveCanBuildFrom[F] =
       new DeriveCanBuildFrom[F] {
+        @SuppressWarnings(Array("scalafix:DisableSyntax.asInstanceOf"))
```

Comment:
Personally I'm not sure how much value there is in using settings like this within this library.

I can see this being useful when you are trying to introduce more functional paradigms into an organization that doesn't currently follow them and want to just disallow a lot of things that people do by default. But I think we are pretty far away from that.

We already disfavor things like `asInstanceOf` but they can be useful in certain cases for allowing us to express things we know to be true but the compiler can't prove or for efficiency to for example avoid mapping implicit evidence over a deeply nested data structure.

Currently if you use `asInstanceOf` you can do it but it is definitely going to raise an eyebrow in a code review and with this you can suppress the warning and that is going to raise an eyebrow in a code review. I think my suggestion would be to focus these rules on things that are for at least practical purposes "always wrong" (e.g. automatic tupling).

Reply:
oh, sure! these two were actually leftovers, because the belong to Scala != 2.12, so Scalafix didn't notify me about these unused suppressions and so I haven't removed them :wink: